examples/mb/model_builders_xgboost_shap.py

```diff
@@ -0,0 +1,80 @@
+# ==============================================================================
```

Comment:
Will example from mb directory actually run?  I think new directory should be added to examples launcher script